wandb-scim/users.py

```diff
@@ -145,16 +132,15 @@ def activate(self, user_id):
                 }
             ],
         }
-        response = requests.patch(url, headers=headers, json=payload)
-
-        if response.status_code == 200:
-            return "User activated successfully!"
-        elif response.status_code == 404:
-            return "User not found"
-        else:
+        try:
+            response = requests.patch(url, json=payload, **self.request_kwargs)
+        except requests.HTTPError:
+            if response.status_code == 404:
+                return "User not found"
             return f"Failed to activate user. Status code: {response.status_code}"
+        return "User activated successfully!"
```

Comment:
```suggestion
        return "User activation succeeded"
```
There's no need to be so excited about it :)

Reply:
See the existing message -- this intentionally preserves prior behavior

If we want to change the return value, we should probably return something more conventional than a string message (e.g. just `None` or `Self` for further method chaining -- both are both common), but because this changes behavior, it goes well beyond the intended scope of this PR